base/Dockerfile

```diff
@@ -9,7 +9,7 @@ ENV TZ=America/Detroit
 # Install common build tools, sqlite, and tzdata
 RUN apt-get update && apt-get install -y build-essential software-properties-common sqlite3 tzdata
 
-# Install Python and Java (required by html5validator)
+# Install Python, Java (required by html5validator), and file (used in tests)
 RUN add-apt-repository ppa:deadsnakes/ppa && apt-get update
 ENV PYTHON python3.10
-RUN apt-get install -y $PYTHON $PYTHON-venv openjdk-8-jre
+RUN apt-get install -y $PYTHON $PYTHON-venv openjdk-8-jre file
```

Comment:
TIL that this was moved to stand-alone repo.  I think it used to be in coreutils.

Reply:
Yeah, I don't understand why they did that. Traditionally it's shipped with basically every Unix-like OS.